sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -265,36 +257,36 @@
     </item>
     <item>
       <EFDB_Name>ambientTemperature</EFDB_Name>
-      <Description>Ambient Temperature Sensor Telemetry</Description>
+      <Description>Ambient Temperature Sensor Telemetry.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>skyTemperature</EFDB_Name>
-      <Description>Sky Temperature Sensor Telemetry</Description>
+      <Description>Sky Temperature Sensor Telemetry.</Description>
```

Comment:
I can see why you retained ambientTemperature and skyTemperature (though is there any difference between ambient and air -- you use airTemperature for the humidity sensors).

Reply:
I'll leave this as is to be able to distinguish between the two.